### ISSUE TITLE
Delete key requires focusing on canvas before removing selected elements

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#1781]
## Bug Description

Delete key seems to work only if canvas has focus first, resulting it deleting elements sometimes not working as expected.

## Expected Behaviour

Delete key should work on a global level, with taking into consideration active selection or input focus -- in these cases, the Delete key should have default behavior (see the [`nativeCopyPasteExpected.js` ](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/utils/nativeCopyPasteExpected.js) util which could perhaps be renamed to `nativeEventHandlingExpected.js` or sth similar to use it for this occasion, too)

## Steps to Reproduce

1. Select an element (or more)
2. Click on one of the library tabs, e.g. shapes
3. Hit `Delete`.
4. See nothing happening.

## Screenshots

Example on FF where clicking "Delete" key tries to go to the previous browser history entry:
![delete](https://user-images.githubusercontent.com/3294597/81944023-6bbbfa00-95fc-11ea-93a7-1581e8c4703f.gif)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by dreamofabear** [comment]
+1. This also applies for flip image horizontal/vertical.

**Comment by dvoytenko** [comment]
Moving temporarily to "Ready for QA" to try out alternatives on the QA site.

**Comment by samitron7** [comment]
All tickets related to focus, please read below. @dvoytenko will take those on.

@dvoytenko  and I came up with some core principles to hopefully handle focused states between the selected element on the page and the design/element panel for the majority of our use cases. In order to wrangle in complexity and balance accessibility needs, we have distilled things down to the following principles

**1. If the thing your mouse is actively on (ie, button or input field) can accept an action (ie delete, paste, arrow move left/right etc), then the focus goes to that thing and not the element on the page. If the thing can not accept an action then the action goes to the element on the page.**

Examples

    - I click on the B button then hit delete. The element will be deleted
    - I click into the input field and hit delete. The value in the input field is deleted

**2. Arrow keys will act on the element except**

    - When I click on the page thumbnail, I expect to be able to move between pages with the arrow key
    - When I click on the tab navigation in the element panel.  Due to accessibility guidelines we need to allow users to capture arrow key for navigating between tab and between images so the arrow keys will not work on the element. To refocus on the element, users will just need to click on the element. This seems to be a common pattern on other applications.

**3. Focus on an element is lost (but the element will remain selected) for the following case** 

    - I have an element selected and I mouse over and click into the search field in the element panel

**Comment by swissspidy** [comment]
> Moving temporarily to "Ready for QA" to try out alternatives on the QA site.

FWIW from a technical perspective it's not needed to do that in order to be able to test on the QA site.
